charm-slurmd/actions.yaml

```
@@ -97,3 +97,10 @@ singularity-install:
     or .rpm (CentOS) packages retrieved from GitHub Releases.
 
     Note: The .deb or .rpm files must be supplied as Juju resources.
+
+mpi-install:
+  description: >
+    Install MPI (mpich). This might take a few minutes to complete.
+
+    This action will install mpich using the package manager 
+    apt-get (Ubuntu) or yum (CentOS).
```

```suggestion
    Install MPI (`mpich`). This might take a few minutes to complete.
```

We don't need to specify what package manager is used here.